crates/astria-sequencer-relayer/src/relayer/mod.rs

```diff
@@ -124,9 +125,8 @@ impl Relayer {
     /// failed catastrophically (after `u32::MAX` retries).
     #[instrument(skip_all)]
     pub(crate) async fn run(self) -> eyre::Result<()> {
-        let submission_state = read_submission_state(&self.pre_submit_path, &self.post_submit_path)
-            .await
-            .wrap_err("failed reading submission state from files")?;
+        let submission_state_at_startup =
+            SubmissionStateAtStartup::new_from_path(&self.submission_state_path).await?;
```

Comment:
Hmm - I can add it here, but as you suspected, it's already being logged further down the failure chain.  I'm not a fan of repeating the same info since it makes the logs needlessly verbose.

I guess I'd prefer to add a comment in the code justifying the absence of `wrap_err` rather than polluting the logs.  Would that be ok?

Reply:
Yeah, that'd would be ok :)